tests/integration/ceph_mgr_test.go

```diff
@@ -113,9 +113,26 @@ func (s *CephMgrSuite) TearDownSuite() {
 	s.installer.UninstallRook()
 }
 
-func (s *CephMgrSuite) execute(command []string) (error, string) {
-	orchCommand := append([]string{"orch"}, command...)
-	return s.installer.Execute("ceph", orchCommand, s.namespace)
+func (s *CephMgrSuite) execute(command []string, maxRetries int) (error, string) {
```

Comment:
How about defining two methods? `execute()` and `executeWithRetry()`? The first one can call the second one with an argument of 1, but it makes the calling code more readable.
```